parley/src/layout/data.rs

```diff
@@ -195,12 +196,13 @@ pub(crate) struct LayoutData<B: Brush> {
     pub(crate) text_len: usize,
     pub(crate) width: f32,
     pub(crate) full_width: f32,
-    pub(crate) min_content_width: f32,
-    pub(crate) max_content_width: f32,
     pub(crate) height: f32,
     pub(crate) fonts: Vec<Font>,
     pub(crate) coords: Vec<i16>,
 
+    // Lazily calculated values
+    content_widths: OnceCell<(f32, f32)>,
```

Comment:
Minor: I would prefer a struct with named fields here. Or at least a comment explaining what each item in the tuple represents.